plugins/topology.c

```diff
@@ -535,6 +535,107 @@ static struct amount_msat peer_capacity(const struct gossmap *gossmap,
 	return capacity;
 }
 
+/* Check if this channel is with us. If yes, add the channel's
+ * inbound routing info to the json response. */
+static bool json_add_channel_inbound(struct json_stream *js,
+				     u32 node_idx,
+				     struct gossmap_chan *chan)
+{
+	for (int i = 0; i < 2; i++) {
+		if (chan->half[i].nodeidx != node_idx)
+			continue;
+		json_object_start(js, "inbound_channel");
+		json_add_u64(js, "fee_base_msat", chan->half[i].base_fee);
+		json_add_amount_msat(js, "htlc_min_msat",
+				     amount_msat(fp16_to_u64(chan->half[i]
+							     .htlc_min)));
+		json_add_amount_msat(js, "htlc_max_msat",
+				     amount_msat(fp16_to_u64(chan->half[i]
+							     .htlc_max)));
+		json_add_u32(js, "fee_proportional_millionths",
+			     chan->half[i].proportional_fee);
+		json_add_u32(js, "cltv_expiry_delta", chan->half[i].delay);
+		json_object_end(js);
+		return true;
+	}
+	return false;
+}
+
+static struct command_result *json_getremoteliquidity(struct command *cmd,
+						      const char *buffer,
+						      const jsmntok_t *params)
+{
+	struct json_stream *js;
+	struct gossmap *gossmap;
+	const jsmntok_t *nodelist;
+
+	if (!param(cmd, buffer, params,
+		   p_req("nodelist", param_array, &nodelist),
```

Comment:
`nodes` maybe is a better field name? I always feel strange to have this `nodelist` only in one word, also because our codegen is not so smart, and we can have not a great field name in the code